star-rx72n-firmware/tests/test_rx_frame.c

```
@@ -989,6 +998,345 @@ void test_go_compatibility_command_with_payload(void)
+{
+  static const uint8_t expected_wire[k_xc_ping_wire_len] = {
+    0x55, 0xAA,                   /* SYNC */
+    0x00, 0x00,                   /* SEQ=0 */
+    0x00, 0x00,                   /* LEN=0 */
+    0x00,                         /* TYPE=PING */
+    0x00,                         /* FLAGS=none */
+    0xEB, 0xAE, 0x3F, 0x9B       /* CRC-32 LE = 0x9B3FAEEB */
+  };
+  rx_frame_t frame;
+  uint8_t    buffer[k_test_small_buffer];
+  uint32_t   len;
```

Clang (14.0.6)

[warning] variable 'len' is not initialized

(cppcoreguidelines-init-variables)